src/en/news/blog/2025/rbd-live-migration/index.md

```diff
@@ -0,0 +1,478 @@
+
+Note: Linux KRBD kernel clients currently do not support live migration.
+
+### What Can You Migrate?
+
+* Between different storage pools (e.g., moving from a high-performance replicated TLC SSD pool to a cost-effective erasure-coded QLC or HDD pool).
+* Within the same pool (e.g., reorganizing data for better management or to effect compression).
+* Across different formats or layouts (e.g., upgrading from an older storage format to a newer one).
+* From external data sources (e.g., migrating from a non-Ceph storage system into Ceph).
+
+### Live Migration: Import-Only Mode
+
+Want to migrate data from an external source or storage provider? No problem! You can:
+
+* Import data from a backup file
+* Pull data from a web URL (HTTP/HTTPS file)
+* Move data from an S3 storage bucket
+* Connect to an NBD (Network Block Device) export
+
+### How Does Live Migration Work?
+
+When you start a live migration, here’s what happens behind the scenes:
+
+* Deep Copying – The system duplicates the entire image while keeping all historical snapshots.
+* Sparse Data Optimization – Only the actual used data is copied, saving storage space and speeding up the process.
+* Seamless Transition – The migration happens while the image remains usable, minimizing downtime.
+* Source Becomes Read-Only – The original image is locked so no new changes are made.
+* Automatic I/O Redirection – All applications and users automatically start using the new image without interruptions.
+
+### Why Is This Important?
+
+* Keeps data flexible – Move storage based on performance, cost, or organizational needs.
+* Ensures data integrity – Snapshot history and structure remain intact.
+* Works in real-time – Migration happens without disrupting workloads.
+
+### Step-by-Step Guide to Live Migrating Ceph RBD Images 
+
+Live migration of RBD images in Ceph allows you to move storage seamlessly between
+pools, RBD namespaces, and clusters in different formats with minimal downtime.
+Let's break it down into three simple steps, along with the necessary commands to execute them.
+
+#### Step 1: Prepare for Migration
+
+In this step before starting the migration, a new target image is created and
+linked to the source image.
+
+* If import-only mode is not enabled, the source image will be marked as read-only to prevent modifications.
+* Any attempts to read uninitialized parts of the new target image will redirect the read operation to the source image.
+* If data is written to an uninitialized part of the target image, Ceph automatically deep-copies the corresponding blocks from the source.
+
+**Syntax:**
+
+```
+rbd migration prepare SOURCE_POOL_NAME/SOURCE_IMAGE_NAME TARGET_POOL_NAME/TARGET_IMAGE_NAME
+```
+
+**Example:**
+
+```
+rbd migration prepare source_pool1/source_image1 target_pool1/target_image1
+```
+
+We can initiate the import-only live migration process by running
+the ``rbd migration prepare`` command with the ``--import-only`` and
```

Comment:
s/``--import-only``/``--import-only`` flag/

If these command modifiers take arguments, they're options. If they don't take arguments, they're flags.

Also, the sentence doesn't work unless "flag" appears in this first position. Otherwise the sentence is just too cataphoric.